mesonbuild/modules/gnome.py

```diff
@@ -979,24 +947,67 @@ def gdbus_codegen(self, state, args, kwargs):
             else:
                 self._print_gdbus_warning()
                 cmd += ['--generate-c-code', '@OUTDIR@/' + namebase, '@INPUT@']
-            outputs = [namebase + '.c', namebase + '.h']
-            install = kwargs.get('install_header', False)
+
             custom_kwargs = {'input': xml_files,
-                             'output': outputs,
+                             'output': output,
                              'command': cmd,
+                             'build_by_default': build_by_default
+                             }
+
+        base_custom_target = build.CustomTarget(output, state.subdir, state.subproject, custom_kwargs)
```

Comment:
Nitpick: maybe you want to rename this to `cfile_custom_target` and the one afterwards to `hfile_custom_target`, and the docbook one as `docbook_custom_target` so that it's easier to follow what's happening?

Reply:
Just did it :)